.github/workflows/cloud-admin.yml

```diff
@@ -0,0 +1,43 @@
+name: Build and Deploy Cloud Admin to Netlify
+on:
+  push:
+    tags:
+      - "*"
+    branches: ["main"]
+    paths:
```

Comment:
I don't think we need `paths` here – any tag on `main` can just trigger a build.

If you want to keep the `paths` – we also need to add `web-common` here